src/helpers/page-subtitle.js

```diff
@@ -2,7 +2,7 @@
 
 module.exports = (page) => {
   if (page.layout === 'landing' || page.layout === 'training-enrollment') {
-    // since the component title and the page title are often the same, we use 'Neo4j Graph Database Platform' to avoid redundancy.
+    // since the component title and the page title are often the same, we use 'Neo4j Graph Data Platform' to avoid redundancy.
     return ' - Neo4j Graph Database Platform'
```

Comment:
I think this line also needs the Database -> Data change for the pages that match the if condition

Reply:
Good spot - done!